bundle/API/Repository/TagsService.php

```diff
@@ -76,7 +76,7 @@ public function loadTagByUrl(string $url, array $languages): Tag;
      *
      * @return \Netgen\TagsBundle\API\Repository\Values\Tags\TagList
      */
-    public function loadTagChildren(?Tag $tag = null, int $offset = 0, int $limit = -1, ?array $languages = null, bool $useAlwaysAvailable = true): TagList;
+    public function loadTagChildren(?Tag $tag = null, int $offset = 0, int $limit = -1, ?array $languages = null, bool $useAlwaysAvailable = true, ?bool $showHidden = null): TagList;
```

Comment:
Possibly we should have a default value for `$showHidden` here, instead of resolving it in the Persistence implementation. @emodric I'll leave that up to you :)

Reply:
@emodric one question though, if here the default is not `null`, how do we detect explicitly passed value in the siteaccess-aware layer?

Reply:
I would say we do, this is not only for rendering, it might be used in import for example.

Reply:
Import scripts can be run in admin siteaccess if needed.

It's just weird design, is all, since API interface presumes that somwhere down below, the value we give to the method might be overriden.

But okay, if there are valid usecases, I can live with it.

Reply:
What I meant was, it looks like API interface is assuming the implementation, or at least one of them.

In terms of API, we either show hidden tags, or don't show hidden tags, there's no third option. And then along comes the third option, that says: "let the implementation decide", which is kinda ambiguous since the user does not know what the end result will be. That's why I'm saying that this API interface is looking weird.

But no matter, we already decided that this is relatively fine, so lets just move on :)

Reply:
As mentioned f2f, we need to document that `null` is interpreted `true` (or `false`), and then it's kind of OK :)